README.md

```diff
@@ -68,6 +68,46 @@ The collection is also available as `ansible-collection-theforeman-foreman` from
 
 After installing the RPM, you can use the modules in the same way as when they are installed directly from Ansible Galaxy.
 
+## Installation From Source
+
+For development or testing purposes, you can install the collection from sources. For production usage, see the instructions above on installing the latest stable release.
+
+### Installation from Github Repository
+
+With Ansible >= 2.10, you can install from a Github repository (such as this one or your fork):
+
+`ansible-galaxy collection install git+https://github.com/theforeman/foreman-ansible-modules.git`
+
+Using SSH instead of HTTPS, you can do:
+
+`ansible-galaxy collection install git@github.com/theforeman/foreman-ansible-modules.git`
+
+You can also specify a branch to use such as `devel`:
+
+`ansible-galaxy collection install git+https://github.com/theforeman/foreman-ansible-modules.git,devel`
+
+To install from a `requirements.yml` file (useful when installing multiple collections) add a snippet to your `requirements.yml` like
+
+```yml
+---
+collections:
+  - name: https://github.com/theforeman/foreman-ansible-modules.git
+    type: git
+    version: devel
+```
+
+And install all specified requirements with `ansible-galaxy install -r requirements.yml`
+
+### Building and Installing the Collection Locally
+
+For older versions of Ansible (< 2.10) where the above approach is not yet supported, you can build the collection locally:
```

Comment:
```suggestion
For all versions of Ansible (including < 2.10) where the above approach is not yet supported, you can build the collection locally:
```

Technically i think this is limited to >= 2.8 ...

Reply:
Thanks! With the latest update, I've been more clear and specific with language here, and in a few other spots.